### PR TITLE
Update DG

### PR DESCRIPTION
**Pull request by HolmesJJ**
In progress

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -1564,3 +1564,68 @@ testers are expected to do more *exploratory* testing.
...
+
+   A timer thread can be added in the background of the application to keep track of all the reservations of members. When an upcoming reservation is detected, a prompt dialog will be popped up or the relevant member reservation information will be displayed in the upper right corner of the application.
+
+1. **Implement by following the proposed [undo/redo](#undo-redo)**
+
+1. **Add a command shortcut [-pg/] for pagination**
+
+   By default, only 20 records are displayed at a time, the user and a parameter [-pg/] is added to the paging function, so that users can turn pages by typing quickly.
+
+   Continue to implement the current limitations mentioned above to make the program more user-friendly.
```
Comment (zzybluebell) [inline docs/DeveloperGuide.md:1631]: Awesome

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -1508,7 +1508,7 @@ testers are expected to do more *exploratory* testing.
    1. Prerequisites: 
       1. Member wih member ID `00001` is in the member list.
       1. Multiple reservations are in the reservation list of the member with member ID `00001`.
-      1. [Viewing member profile with member ID `00001`](#viewing-member-profile).
+      1. [Showing member profile with member ID `00001`](#showing-member-profile).
 
    1. Test case: `del -rs/ -id/00001000001`<br>
        Expected: Reservation with reservation ID `000001` in the reservation list of the member with member ID `00001` is deleted. Details of the deleted reservation shown in the status message.
```
Comment (zzybluebell) [inline docs/DeveloperGuide.md:1514]: Good

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -1137,25 +1137,25 @@ Manager is logged in
...
 
-* 1c. The program currently stores no more than 30 records, and `up` key is pressed more than the number of stored records continuously from the beginning.
+* 1c. The program currently stores no more than 30 commands history, and `up` key is pressed more than the number of stored commands history continuously from the beginning.
 
     * 1c1. ezFoodie shows empty in the command box.
 
       Use case resumes at step 1.
 
-* 1d. The program currently stores no more than 30 records, and `down` key is pressed more than the number of stored records continuously from the beginning.
+* 1d. The program currently stores no more than 30 commands history, and `down` key is pressed more than the number of stored commands history continuously from the beginning.
```
Comment (zzybluebell) [inline docs/DeveloperGuide.md:1158]: should be "the history of 30 commands?"

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -1137,25 +1137,25 @@ Manager is logged in
...
 
-* 1b. The program currently stores 30 records, and `down` key is pressed more than 30 times continuously from the beginning.
+* 1b. The program currently stores 30 commands history, and `down` key is pressed more than 30 times continuously from the beginning.
 
     * 1b1. ezFoodie shows empty in the command box.
 
       Use case resumes at step 1.
 
-* 1c. The program currently stores no more than 30 records, and `up` key is pressed more than the number of stored records continuously from the beginning.
+* 1c. The program currently stores no more than 30 commands history, and `up` key is pressed more than the number of stored commands history continuously from the beginning.
```
Comment (zzybluebell) [inline docs/DeveloperGuide.md:1152]: same?

File: docs/DeveloperGuide.md [hunk 5]
```diff
@@ -1137,25 +1137,25 @@ Manager is logged in
 
 **Extensions**
 
-* 1a. The program currently stores 30 records, and `up` key is pressed more than 30 times continuously from the beginning.
+* 1a. The program currently stores up 30 commands history, and `up` key is pressed more than 30 times continuously from the beginning.
```
Comment (zzybluebell) [inline docs/DeveloperGuide.md:1140]: same?

**Review by mukundrs — COMMENTED** [review]
I agree with Yuzhao's points. Apart from that everything else looks good!

**Review by mukundrs — APPROVED** [review]
LGTM!

**Pull request merged**